internal/sink/partition.go

```diff
@@ -0,0 +1,23 @@
+package sink
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/confluentinc/confluent-kafka-go/v2/kafka"
+)
+
+func prettyPartitions(partitions []kafka.TopicPartition) []string {
+	out := []string{}
+
+	for i := range partitions {
+		topicName := ""
+		if partitions[i].Topic != nil {
+			topicName = *partitions[i].Topic
+		}
+		out = append(out, fmt.Sprintf("%s-%d", topicName, partitions[i].Partition))
+	}
+
+	sort.Strings(out)
+	return out
```

Comment:
```suggestion
	out := make([]string, 0, len(partitions))

	for _, partition := range partitions {
		var topicName string
		
		if partition.Topic != nil {
			topicName = *partition.Topic
		}
		
		out = append(out, fmt.Sprintf("%s-%d", topicName, partition.Partition))
	}

	sort.Strings(out)
	
	return out
```

Reply:
What if the topic is empty? Is (missing topic) "-1" valid?

Reply:
It's unclear why the topic can be nil: https://github.com/confluentinc/confluent-kafka-go/blob/27274c264f39cdf429a88a762f9796f36837f7b0/kafka/kafka.go#L279

But as this method is only used to pretty logs it's not a problem.